packages/ibm-products/src/global/js/hooks/useFocus.js

```diff
@@ -85,40 +85,27 @@ export const useFocus = (modalRef, selectorPrimaryFocus) => {
     }
   };
 
+  const claimFocus = () => {
```

Comment:
Should this be wrapped in a useCallback so we can add it to the dep arrays in our useCallbacks above? If we do that we could potentially lift all of the `eslint-disable-next-line react-hooks/exhaustive-deps` rules